SimExLite/PropagationCalculators/WPGProgagationCalculator.py

```diff
@@ -0,0 +1,112 @@
+""":module WPGPropagationCalculator: Module that holds the WPGPropagationCalculator class."""
```

Comment:
Module filename: should be WPGPropagationCalculator.py, not ~WPGProgagationCalculator.py~

Reply:
That's a super embarrassing mistake. Done.